mindmeld/models/helpers.py

```diff
@@ -293,6 +294,22 @@ def get_ngram(tokens, start, length):
     return " ".join(ngram_tokens)
 
 
+def get_ngrams_upto_n(tokens, n):
+    """This function returns a generator that returns ngram tuples with length upto n
+
+    Args:
+        tokens (list of str): Word tokens.
+        n (int): The length of n-gram upto which the ngram tokens are generated
+
+    Returns:
+        tuple: ngram, (token index start, token index end)
+    """
+    assert n > 0
+    for length, i in enumerate(range(1, n + 1)):
+        for idx, j in enumerate(nltk.ngrams(tokens, i)):
```

Comment:
Curious why you decided to use this as opposed to implementing it?

Reply:
it returns a tuple of ngrams instead of a concatenated string. also since we already imported nltk